agent/bench-scripts/pbench-uperf

```diff
@@ -369,12 +381,29 @@ while true; do
 		;;
 	esac
 done
+
+if [[ ${do_pre_check_only} -eq 1 ]]; then
```

Comment:
OK, the difference in option handling between fio and uperf suddenly became vaguely annoying: that y/n `[[ "${pre_check}" == "y" ]]` there vs `[[ ${do_pre_check_only} -eq 1 ]]` here. I know, other options are handled differently, too, and making them consistent is beyond the scope of this PR... but you made me read it again, so it's your fault!

Reply:
Ha!

Yes, I am painfully aware of that.  And have scars on my hands from working with this code.

But ... it is out of the scope of this PR, and really this kind of things is best handled in a re-write or just by dropping these commands in favor of `benchmark-wrapper`.